### ISSUE TITLE
Purchases: Certain texts on expired upgrades are hard to read

### ISSUE THREAD

**Issue by hafizrahman** [Automattic/wp-calypso#258]
On https://wordpress.com/purchases:
![screen shot 2015-11-20 at 4 50 08 pm](https://cloud.githubusercontent.com/assets/266376/11296950/c2b58d9a-8fa7-11e5-83bc-602764efb233.png)

On On https://wordpress.com/purchases/:site/:purchaseId
![screen shot 2015-11-20 at 4 50 25 pm](https://cloud.githubusercontent.com/assets/266376/11296949/c2b0619e-8fa7-11e5-8a9b-cbfc73a439ec.png)

From the CSS it looks like there's an added opacity of 0.6 to these texts. I don't see why it's a good idea to make an expired upgrade's information to be harder to read. An expired upgrade can affect a user's site negatively in various ways. Wouldn't it be better if we present that information clearer?

Here's the WCAG color contrast checker result for the least contrast-y text color on those screenshots: http://webaim.org/resources/contrastchecker/?fcolor=668eaa&bcolor=f3f6f8

> Normal Text
> WCAG AA: Fail
> WCAG AAA: Fail
> Large Text
> WCAG AA: Pass
> WCAG AAA: Fail

**Comment by fabianapsimoes** [comment]
Pinging @mikeshelton1503 for feedback on Marjorie's findings.

**Comment by mikeshelton1503** [comment]
So the colors used here are consistent with other areas of Calypso so I think this should be brought up a broader scale. Let's leave this open for now and I'll use it as a basis to start the conversation.

**Comment by drw158** [comment]
Aside — can we make sure that the color Marjorie is referring to is using a color variable? The one she tested is not in our handbook I don't think. Doesn't affect the accessibility though, I just wanted to make sure.

**Comment by mikeshelton1503** [comment]
>can we make sure that the color Marjorie is referring to is using a color variable? The one she tested is not in our handbook I don't think.

Yep it's using `gray`. The color checker must have just sample it slightly incorrectly.

**Comment by fabianapsimoes** [comment]
@mikeshelton1503 did anything come out of it, or should we call this a won't fix?

**Comment by mikeshelton1503** [comment]
No nothing has been started in this area AFAIK, however we don't need to keep this one open. Any change here will need a broader effort.

**Comment by fabianapsimoes** [comment]
Thanks, Mike! I'll close this one then.
